src/components/activities/RosterPanel.tsx

```diff
@@ -86,7 +86,9 @@ export function ParticipantDialog({ open, participant, activity, onClose }: { op
                   <TableRow key={t.time}>
                     <TableCell>{activity.organizations[t.organizationId].rosterName ?? activity.organizations[t.organizationId].title}</TableCell>
                     <TableCell>{getStatusText(t.status)}</TableCell>
-                    <TableCell>{formatDate(t.time, 'EEE yyyy-MM-dd HHmm')}</TableCell>
+                    <TableCell>
+                      <OutputTime label="" time={t.time} relative={false}></OutputTime>
```

Comment:
Not a fan of using the Output Form Fields like this. They apply unnecessary styling, here.

This could just be a RelativeTimeText:
```
<TableCell>
  <RelativeTimeText time={time} baseTime={new Date().getTime()} />
</TableCell>
```

Reply:
Was there different styling that happened? I didn't notice it if so. After merging your changes, there's actually no need to use RelativeTimeText here at all anyway. I see that you're using a different DateTimePicker for the roster edit versus what we have in the update status dialog. We should probably be consistent with which DateTimePicker we use.